### PR TITLE
Various fixes.

### PR DESCRIPTION
**Pull request by oliverchang**
- Fix an incorrect indent leading to multiple packages not being output
  correctly in the JSON.
- Fix aliases/related parsing.

### CODE REVIEW

**Comment by oliverchang** [comment]
tbr

**Review by inferno-chromium — COMMENTED** [review]
LGTM

File: lib/osv/repos.py [hunk 1]
```diff
@@ -43,6 +43,7 @@ def __init__(self, username, ssh_key_public_path, ssh_key_private_path):
     self.ssh_key_private_path = ssh_key_private_path
 
   def credentials(self, url, username_from_url, allowed_types):
+    """Get credentials."""
```
Comment (jonathanmetzman) [inline lib/osv/repos.py:46]: nit: "Gets credentials" to follow styleguide.

**Comment by jonathanmetzman** [comment]
LGTM

**Pull request merged**